lib/Service/SettingService.php

```diff
@@ -0,0 +1,184 @@
+<?php
```

Comment:
Consider naming it `SettingsService`. Not sure why, but "settinS" is usually plural in English 🤷‍♂

Reply:
You're right.. :sweat_smile: